### DISCUSSION TITLE
[Enhancement] [tree-v2] TreeV2 像 Tree一样支持拖拽？

### DISCUSSION THREAD

**Discussion by einsitang** [element-plus/element-plus#9303] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `tree-v2`

## Description

TreeV2 相比于 Tree 快多了，但是我想实现和Tree一样的拖拽功能，请问可以支持吗

### Additional comments

_(empty)_

**Comment by wu529778790** [comment]
虚拟列表的拖拽，拖到刚开始没有渲染出来的dom上的时候，并不会触发，拖拽事件

**Comment by einsitang** [comment]
的确不好满足这样的feature

**Comment by weixgit** [comment]
el-tree数据量大时拖拽卡死，有没其他方案可以解决
